src/base.js

```diff
@@ -13,25 +13,56 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import os from 'os';
 import { Command, flags as flagParser } from '@oclif/command';
+import { getConfig } from './utils/config';
+import print from './utils/print';
+
+export const defaultConfigFolder = '.lisk-commander';
 
 export default class BaseCommand extends Command {
 	async init() {
 		const { flags } = this.parse(this.constructor);
 		this.flags = flags;
-	}
 
-	async catch(error) {
-		this.log({ error });
+		process.env.XDG_CONFIG_HOME =
+			process.env.LISK_COMMANDER_CONFIG_DIR ||
+			`${os.homedir()}/${defaultConfigFolder}`;
+		this.userConfig = getConfig(process.env.XDG_CONFIG_HOME);
+		this.printOptions = Object.assign(
+			{},
+			{
+				json: this.userConfig.json,
+				pretty: this.userConfig.pretty,
+			},
+			this.flags,
+		);
```

Comment:
Since we have node 8 in v2 branch, from now on I think we should use object spread and stick with it.